test/e2e/page-objects/flows/transaction-redesign.flow.ts

```diff
@@ -0,0 +1,54 @@
+import { DAPP_URL, WINDOW_TITLES } from '../../helpers';
+import { Driver } from '../../webdriver/driver';
+import TestDapp from '../pages/test-dapp';
+import Dialog from '../pages/dialog';
+
+export const createERC721SetApprovalForAllTransaction = async (
+  driver: Driver,
+  contractAddress: string,
+) => {
+  console.log('Create ERC721 setApprovalForAll transaction');
+
+  const testDapp = new TestDapp(driver);
+
+  await testDapp.open({ contractAddress, url: DAPP_URL });
+
+  await testDapp.clickERC721SetApprovalForAllButton();
```

Comment:
Since these flows are so simple and have no configuration, would it actually improve readability and maintenance of the tests themselves if we just included this code directly in the tests?